### PR TITLE
Add logo usage to handbook

### PR DESCRIPTION
**Pull request by Yndira-FlowForge**
## Description

Add a section about the logo in handbook/design/branding/

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Review by ZJvandeWeg — APPROVED** [review]

File: src/handbook/design/branding.md [hunk 1]
```diff
@@ -1,6 +1,120 @@
...
+</div>
+
+#### Vertical Version
+
+<hr style="margin-bottom:30px; border-color:#E5E7EB;">
+<div class="flex h-full flex-col justify-center items-center md:mt-0 mb-14">
+    <img width="180" min-width="180" alt="Image of the vertical version of FlowFuse logo" src="../images/logos/ff-logo--square--light.png">
+</div>
+
+### Minimun Sizes
```
Comment (joepavitt) [inline src/handbook/design/branding.md:28]:
```suggestion
### Minimum Sizes
```

Comment (Yndira-FlowForge) [reply src/handbook/design/branding.md:28]: Is there a difference?

Comment (joepavitt) [reply src/handbook/design/branding.md:28]: `Minimun` (with an 'n' at the end) isn't a word :)

Comment (Yndira-FlowForge) [reply src/handbook/design/branding.md:28]: Ohhh I see now!! :ok_hand: :ok_hand:

**Review by joepavitt — APPROVED** [review]

**Pull request merged**